src/khoj/database/adapters/__init__.py

```diff
@@ -391,6 +393,54 @@ async def aget_client_application_by_id(client_id: str, client_secret: str):
         return await ClientApplication.objects.filter(client_id=client_id, client_secret=client_secret).afirst()
 
 
+class AgentAdapters:
+    DEFAULT_AGENT_NAME = "khoj"
+    DEFAULT_AGENT_AVATAR = "https://khoj-web-bucket.s3.amazonaws.com/lamp-128.png"
+
+    @staticmethod
+    async def aget_agent_by_id(agent_id: int):
+        return await Agent.objects.filter(id=agent_id).afirst()
```

Comment:
Should we add check here to see if `user_id` is permitted to access the specified agent_id (i.e if `agent_id` is either public or created by user_id)? This will add a little more in-depth defense against unauthorized access to private agents even if you know that agent's id